pkg/cephmgr/cephd/cephd.go

```diff
@@ -164,6 +165,7 @@ func (c *ceph) Run(id string, args ...string) error {
 		ret = C.cephd_run_rgw(C.int(len(finalArgs)), (**C.char)(ptr))
 	} else if id == "rgw-admin" {
 		ret = C.cephd_run_rgw_admin(C.int(len(finalArgs)), (**C.char)(ptr))
+		C.fflush(C.stdout) // Without this long output might get cut off as the process exists before fulling flushing stdout
```

Comment:
is this flush not needed for other calls in this file that rely on stdout? (or are there none?)